cylc/flow/cfgspec/workflow.py

```diff
@@ -401,7 +401,7 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     # differentiate between not set vs set to empty
                     default = None
                 elif item.endswith("handlers"):
-                    desc = desc + '\n\n' + dedent(rf'''
+                    desc = desc + '\n\n' + dedent(f'''
```

Comment:
I'm not super happy about this - will have another look on Monday.

Reply:
This was not flagged by flake8, though?

Reply:
Not directly, but I had a flake8 thing complaining about the backslashes: https://github.com/cylc/cylc-flow/actions/runs/6666089503/job/18116929271

Reply:
Discussed in person - I'm happy with this change